assets/files.wxs

```diff
@@ -1930,6 +1930,11 @@
             <File Id="filFF877991123D49A58FD15CB83AD44BA7" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
Validate New-User.Can create New-User with explicit properties in local machine
```powershell
Expected strings to be the same, but they were different.
Expected length: 11
Actual length:   5
Strings differ at index 0.
Expected: 'TestUserNew'
But was:  'samq2'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 51
51:             $result.Name | Should -BeExactly TestUserNew
```
Validate New-User.Error on user exists
```powershell
A parameter cannot be found that matches parameter name 'BeTypeOf'.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 113
113:             $exc.Exception | Should -BeTypeOf "System.DirectoryServices.AccountManagement.PrincipalExistsException"

```